### DISCUSSION TITLE
Feature Request: Exclude rejects and requests from Trending / Popular / Upcoming

### DISCUSSION THREAD

**Discussion by RichardUUU** [sct/overseerr#1763] [General]
Once I've Requested things, showing them in Trending / Popular / Upcoming is a waste of space and time as I scroll past them.

It's even worse with things I've Rejected because once I figure out that Ambiguously-Named-Show isn't for me, I probably won't remember that name months from now when it's still taking up space and time.

Overseerr would be 10x more useful if it had a Reject button, and a place to see all of my Rejects in case I change my mind.

Cheers, Richard

**Comment by capu57** [comment]
I second this request.  I would also like the ability to reject items on the discovery list so I never see it again.

Not trying to piggy back off this request but I would also like the ability to reject Anime completely.  I do not watch it and there is so much of it than when I scroll through Discovery practically every 3rd item is an Anime.

**Reply by RichardUUU** [reply]
Great idea. Probably just need a genre filter to get rid of everything we don't like.
